src/gui/rss/automatedrssdownloader.ui

```diff
@@ -209,7 +209,7 @@ Supports the formats: S01E01, 1x1, 2017.12.31 and 31.12.2017 (Date formats also
                </sizepolicy>
               </property>
               <property name="text">
-               <string>Assign Category:</string>
+               <string>Category:</string>
```

Comment:
Why do you change it? It isn't category of rule itself. It is Category that will be **assigned** to added torrents. Do you believe that such verbosity is unnecessary?

Reply:
@glassez The main reason was because after adding the icon I felt there wasn't enough horizontal space for the dropdown. I attached a screenshot with the original label. Perhaps shortening may not have been necessary. 
<img width="613" alt="Untitled" src="https://user-images.githubusercontent.com/26905303/106708827-5cc11180-65a8-11eb-8c23-8950ba1d1e5d.png">

Reply:
OK, I think everyone will understand what we are talking about.